src/utils/hints/tokensHints.js

```diff
@@ -0,0 +1,5 @@
+export const tokenHints = {
+  tokenSymbol: 'Symbol of a token. Symbol does not have to be unique.',
```

Comment:
"The symbol does not..."

Reply:
Right, I will spellcheck separately next time .)